### DISCUSSION TITLE
How to Add Input Text fields instead of Read Only Data

### DISCUSSION THREAD

**Discussion by DmitriyMalayev** [aerialist7/kotlin-react-table-sample#13] [Q&A]
Hi I am new to Kotlin and love this repository. I'd like to know how I can add a text fields per each column. Any help appreciated.

**Comment by turansky** [comment]
For common questions like this please use [official documentation](https://tanstack.com/table/v8/docs/examples/react/editable-data)
For more detailed questions you can use fork of this repository
